src/main/java/core/basesyntax/service/impl/ReturnAction.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ActionHandler;
+
+public class ReturnAction implements ActionHandler {
+    public void apply(String fruit, int quantity) {
```

Comment:
The 'apply' method should have @Override annotation because it implements a method from the 'ActionHandler' interface.